data-prepper-plugins/peer-forwarder/src/test/java/com/amazon/dataprepper/plugins/prepper/peerforwarder/discovery/AwsCloudMapPeerListProviderTest.java

```diff
@@ -46,7 +46,7 @@
 
 class AwsCloudMapPeerListProviderTest {
 
-    public static final int WAIT_TIME_MULTIPLIER_MILLIS = 1200;
+    public static final int WAIT_TIME_MULTIPLIER_MILLIS = 120000;
```

Comment:
Thanks for catching this intermittent failure!

This goes from 1.2 seconds to 2 minutes. That could be a long time to wait for a test to fail. Is there value in between that might be more appropriate?